core/setup.cfg

```diff
@@ -61,12 +61,21 @@ zip_safe = False
 
 [options.extras_require]
 dev =
+    coverage[toml]~=6.6.0b1
     pre-commit
-    tox
-test =
     pytest
     pytest-tldr
-    coverage[toml]
+    setuptools-scm[toml]~=7.0
+    tox
```

Comment:
I've added setuptools-scm to the existing core[dev] extra. There was also an existing core[test] extra , but I don't think anything was using it so I've just merged it into core[dev].

Reply:
Yeah - the line between a dev environment and a test environment is pretty faint, so the simplification makes sense.